airbyte_ui_rock/rockcraft.yaml

```diff
@@ -0,0 +1,75 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: airbyte-ui
+summary: Airbyte UI rock
+description: Airbyte UI OCI image for the Airbyte UI charm
+version: "1.0"
+base: ubuntu@22.04
+build-base: ubuntu@22.04
+license: Apache-2.0
+platforms:
+  amd64:
+
+services:
+  airbyte-webapp:
+    override: replace
+    summary: "airbyte-webapp service"
+    startup: disabled
+    command: "/usr/bin/pnpm -C airbyte-webapp start"
```

Comment:
question: any reason this needs to be executed by the root user? And not set up permissions as outlined here: https://discourse.ubuntu.com/t/unifying-user-identity-across-snaps-and-rocks/36469

Reply:
I can't think of a specific reason. Once I set permissions, would I run it as `/bin/pnpm`?

Reply:
You'd run it the same way, it would just be run by the daemon user as opposed to root.

Reply:
As per MM conversation, this seems to have introduced a permission issue with the staged binaries. Will defer for now.